odoo/addons/base/ir/ir_qweb/ir_qweb.py

```diff
@@ -342,7 +342,7 @@ def _get_field(self, record, field_name, expression, tagName, field_options, opt
         field = record._fields[field_name]
 
         # adds template compile options for rendering fields
-        for k, v in options.items():
+        for k, v in filter(lambda x: not x[0].startswith("__"), options.items()):
```

Comment:
why not just add a 3rd line, if not k.startswith('__'):

seems more comprehensive no ;) ?